src/Components/CreateImageWizard/steps/ImageOutput/TargetEnvironment.tsx

```diff
@@ -49,6 +50,11 @@ const TargetEnvironment = () => {
 
   const dispatch = useAppDispatch();
   const prefetchSources = provisioningApi.usePrefetch('getSourceList');
+  const prefetchActivationKeys = rhsmApi.usePrefetch('listActivationKeys');
+
+  useEffect(() => {
```

Comment:
I was trying to figure out whether we needed a useEffect or not for this. The RTKQ docs seem to use a similar pattern for this scenario so I think it makes sense!
https://redux-toolkit.js.org/rtk-query/usage/prefetching#recipe-prefetch-immediately